arrow-avro/Cargo.toml

```
@@ -59,6 +59,8 @@ strum_macros = "0.27"
 uuid = "1.17"
 indexmap = "2.10"
 rand = "0.9"
+md5 = "0.8.0"
+sha2 = "0.11.0-rc.0"
```

Also we should probably go with a stable `sha2` version.

```suggestion
sha2 = "0.10"
```